external-dependencies/download-tibco.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+if [ -f "tibjms.jar" ]; then exit 0; fi
+rm -rf /tmp/tibco
```

Comment:
nit: maybe `mktemp -d tibco-XXX` ? 
(just in case there are few build processes running on the same machine)

Reply:
really? isn't each build in its own container?

Reply:
I don't know. maybe @eloycoto knows?